code/src/index.css

```diff
@@ -1,13 +1,146 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, "Helvetica Neue",
+        sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    background-color: black;
+    color: white;
+    overflow-x: hidden;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.album-name a {
+    font-size: 14px;
+    font-weight: bold;
+}
+
+.artist-container {
+    font-size: 14px;
+    color: #a0a0a0;
+}
+
+.outer-wrapper {
+    display: grid;
+    grid-template-columns: auto;
+    width: 100vw;
+}
+
+header {
+    align-self: center;
+}
+
+.inner-wrapper {
+    width: 80vw;
+    max-width: 900px;
+    padding: 2%;
+    align-self: center;
+    justify-self: center;
+}
+
+.album-container {
+    display: grid;
+    grid-template-columns: auto;
+    row-gap: 5vh;
+}
```

Comment:
Maybe add a column gap (if possible) to create a bit of space between the albums